library/src/scss/mixins/_mixin.breakpoints.scss

```diff
@@ -0,0 +1,28 @@
+/*!
```

Comment:
These are specific to the rich editor, they should be with the rich editor styles. Also prefix the name of the file with richEditor